sdk/route/src/automatic.ts

```diff
@@ -235,6 +239,87 @@ export class NttAutomaticRoute<N extends Network>
     };
   }
 
+  async resume(tx: TransactionId): Promise<R> {
+    const vaa = await this.wh.getVaa(
+      tx.txid,
+      "Ntt:WormholeTransferStandardRelayer"
+    );
+    if (!vaa) throw new Error("No VAA found for transaction: " + tx.txid);
+
+    const msgId: WormholeMessageId = {
+      chain: vaa.emitterChain,
+      emitter: vaa.emitterAddress,
+      sequence: vaa.sequence,
+    };
+
+    const { payload } = vaa.payload;
+    const { recipientChain, trimmedAmount } =
+      payload["nttManagerPayload"].payload;
+
+    const token = canonicalAddress({
+      chain: vaa.emitterChain,
+      address: payload["nttManagerPayload"].payload.sourceToken,
+    });
+    const manager = canonicalAddress({
+      chain: vaa.emitterChain,
+      address: payload["sourceNttManager"],
+    });
+    const whTransceiver =
+      vaa.emitterChain === "Solana"
+        ? manager
+        : canonicalAddress({
+            chain: vaa.emitterChain,
+            address: vaa.emitterAddress,
+          });
+
+    const dstInfo = NttRoute.resolveDestinationNttContracts(
+      this.staticConfig,
+      {
+        chain: vaa.emitterChain,
+        address: payload["sourceNttManager"],
+      },
+      recipientChain
+    );
+
+    const amt = amount.fromBaseUnits(
+      trimmedAmount.amount,
+      trimmedAmount.decimals
+    );
+
+    return {
+      from: vaa.emitterChain,
+      to: recipientChain,
+      state: TransferState.Attested,
+      originTxs: [tx],
+      attestation: {
+        id: msgId,
+        attestation: vaa,
+      },
+      params: {
```

Comment:
out of scope for this change but wdyt about requiring all `Receipt` types to include the `Vp` in the future? NTT is the only route that does this today, I believe.

Reply:
I don't see why we couldn't, I suppose the more information on the receipt the better